frontend/src/lib/components/TaxonomicFilter/types.ts

```diff
@@ -24,7 +24,7 @@ export interface TaxonomicFilterLogicProps extends TaxonomicFilterProps {
 
 export interface TaxonomicFilterGroup {
     name: string
-    type: TaxonomicFilterGroupType
+    type: TaxonomicFilterGroupType | string
```

Comment:
so this kind of type change is problematic because we're losing most validation here. What's the problem you're running into here? Can it be solved by modifying `TaxonomicFilterGroupType`?

Reply:
The biggest problem I'm running into here is that because the group types are dynamic, I'm not sure how to set that in the typing besides leaving it as a `string` option

Reply:
Since we have limited amount of groups (5) we could hard-code them into the GroupType.